spec/betterc.dd

```diff
@@ -81,8 +81,34 @@ $(H2 $(LNAME2 retained, Retained Features))
     $(LI Interfacing with C++)
     $(LI COM classes and C++ classes)
     $(LI `assert` failures are directed to the C runtime library)
+    $(LI `unittest`)
     )
 
+$(H3 $(LNAME2 unittests, Running unittests in `-betterC`))
+
+While, testing can be done without the $(TT -betterC) flag, it is sometimes desirable to run the testsuite in `-betterC` too.
+`unittest` blocks can be listed with the $(DDSUBLINK spec/traits, getUnitTests, `getUnitTests`) trait:
+
+---
+unittest
+{
+   assert(0);
+}
+
+extern(C) void main()
+{
+    static foreach(u; __traits(getUnitTests, __traits(parent, main)))
+        u();
+}
+---
+
+$(CONSOLE
+> dmd -betterC -unittest -run test.d
+dmd_runpezoXK: foo.d:3: Assertion `0' failed.
+)
+
+However, in `-betterC` `assert` don't use Druntime's assert and are directed to `assert` of the C runtime library instead.
```

Comment:
don't --> doesn't, are --> is

Unless you wan to change `assert` to `assert`s.